src/ingest_validation_tools/directory_validator.py

```diff
@@ -4,6 +4,40 @@
 from importlib import import_module
 
 
+def default_factory(base_path, assay_type):
+    """
+    default validator factory
+    """
+    return DefaultDirectoryContentValidator(base_path, assay_type)
+
+
+DIRECTORY_CONTENT_VALIDATOR_FACTORY = default_factory
+
+
```

Comment:
```suggestion
def directory_content_validator_factory(base_path, assay_type):
    """
    default validator factory
    """
    return DefaultDirectoryContentValidator(base_path, assay_type)


```
... and the corresponding change below, if you feel like it? I guess the original name makes me think you would add more factories over time... but I think the advantage of the Factory pattern would be hiding the details inside here, instead.

Reply:
Agreed on both counts.  The problem with providing an example is that I plan to do it in the context of the larger piece of code which imports ingest-validation-tools, but I could add an explicit example.  I tested it by modifying ingest_validation_tools/validation_utils to overlay a substitute factory method at the time of import.  I could add an example to demonstrate that overlay but it'll maybe be more complex than one wants an example to be.  I thought the pattern was common enough that a specific example wouldn't be needed.